scripts/RenameFiles.bat

```diff
@@ -0,0 +1,185 @@
+	echo                  ^|      ___________________________________________________      ^|
+	echo                  ^|                                                               ^|
+	echo                  ^|                ADOBE FILES ARE ALREADY RENAMED!               ^|
+	echo                  ^|                                                               ^|
+	echo                  ^|             Would you like to restore those files?            ^|
+	echo                  ^|      ___________________________________________________      ^|
+	echo                  ^|                                                               ^|
+	echo                  ^|      [1] Restore Adobe files                                  ^|
+	echo                  ^|      ___________________________________________________      ^|
+	echo                  ^|                                                               ^|
+	echo                  ^|      [Q] Exit Module                                          ^|
+	echo                  ^|                                                               ^|
+	echo                  ^|                                                               ^|
+	echo                  ^|_______________________________________________________________^|
+	echo:          
+	choice /C:1Q /N /M ">                                            Select [1,Q]: "
+	if errorlevel 2 (
+		goto exit
+	)
+	if errorlevel 1 (
+		goto mainScript
+	)
+) else (
+	goto targetCheck
+)
+
+:: Check if target path exists
+:targetCheck
+for %%a in (%files%) do (
+	if exist %%a (
+		set targetExists=true	
+	)
+)
+
+if %targetExists% == true (
+	goto mainScript
+) else (
+	cls
+	echo The target file cannot be found. Cannot proceed with renaming adobe files.
+	pause
+	goto exit
+)
+
+:exit
+start cmd /k %~dp0\..\CCStopper.bat
+exit
+
+:mainScript
+cls
+:: Thanks https://github.com/massgravel/Microsoft-Activation-Scripts for the UI
+echo:
+echo:
+echo                   _______________________________________________________________
+echo                  ^|                                                               ^| 
+echo                  ^|                                                               ^|
+echo                  ^|                            CCSTOPPER                          ^|
+echo                  ^|                         Made by eaaasun                       ^|
+echo                  ^|                        RenameFiles Module                     ^|
+echo                  ^|      ___________________________________________________      ^|
+echo                  ^|                                                               ^|
+echo                  ^|                  THIS WILL RENAME ADOBE FILES!                ^|
+echo                  ^|                                                               ^|
+echo                  ^|      It is HIGHLY recommended to create a system restore      ^|
```

Comment:
Creating a system restore point will not backup the targeted files here

Reply:
Added option for backing up adobe files instead of creating restore point